sdcm/utils/common.py

```diff
@@ -1071,3 +1071,16 @@ def get_tables_columns_list(entity_type):
             if 'counter' in column_types:
                 avaialable_ks_cf.pop(ks_cf)
     return avaialable_ks_cf.keys()
+
+
+def format_timestamp(timestamp):
+    return datetime.datetime.utcfromtimestamp(timestamp).strftime('%Y-%m-%d %H:%M:%S')
+
+
+def makedirs(path):
+    try:
```

Comment:
what are you trying to solve here?

It raises an error exception if the leaf directory already exists or cannot be created. 
https://docs.python.org/2/library/os.html#os.makedirs

Reply:
There are 2 cases here:
- if it cannot be created there is nothing to do (the path won't exist)
- if it already exists, `makedirs` succeed <- maybe you wanted to solve this?

Reply:
The idea was to remove duplicating code in several places where several dirs have to be created. I wanted to put this in one place and emulate the functionality from python3 version of os.makedirs ( it has in python3 the new flag exists( if exists then do not throw exception) 
In out code, in several places, if folder is already exists, exception is thrown, and test/teardown or process is interrupted.

Reply:
the problem is in the logic (generalization is OK):
Your code does:
tries to create directory
as I commented: 
if the leaf directory is created -> everything is OK.
if it wasnt' created exception is thrown.
Question: why do you check if the path exists when Python throws exception that the directory can't be created?